src/commands/admin/jobs/check-interval.js

```diff
@@ -0,0 +1,40 @@
+const { processCmd, secsToDHMS, getInlineCodeblock } = require('../../../utils/helper')
+const jobSchema = require('../../../schemas/jobs')
+
+module.exports = ({
+  meta: {
+    name: 'check-interval',
+    description: 'Check the interval for job',
+    commands: ['checkinterval'],
+    permissions: ['ADMINISTRATOR']
+  },
+  run: async (client, cache, message) => {
+      const { args } = processCmd(message)
+      const { guild } = message
+      const jobName = args.shift() || null
+  
+      if (!jobName) {
+        return await message.lineReply('Please specify a Job name: \n```\n' + Array.from(client.jobs.keys()).join('\n') + '\n```')
+      }
+  
+      if (!Array.from(client.jobs.keys()).includes(jobName)) {
+        return await message.lineReply('Invalid Job. Valid Jobs: \n```\n' + Array.from(client.jobs.keys()).join('\n') + '\n```')
+      }
+
+      const job = client.jobs.get(jobName)
+      if (!job.meta.guildControlled) {
+        return await message.lineReply('Guild controlled jobs run on an interval that cannot be changed.')
```

Comment:
This is probably a typo from me, but could this read:
`This job is not guild controlled so the interval cannot be changed.`